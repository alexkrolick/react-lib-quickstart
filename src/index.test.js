import React from 'react'
import { create as render } from 'react-test-renderer'
import Foo from './index'

it('renders', () => {
  const component = render(<Foo />)
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
})