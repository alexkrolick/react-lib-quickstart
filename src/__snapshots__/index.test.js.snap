// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders 1`] = `
<span>
  foo
</span>
`;
